### ISSUE TITLE
Auction not working

### ISSUE THREAD

**Issue by SkunkGuru** [roncli/LibWowAPI#2]
I just downloaded and setup the library. It works fine for characters, but now I have an error reading Auctions. Should the library work with Draenor or is broken / not updated ?
I attach the error and the code where it happens

![exception](https://cloud.githubusercontent.com/assets/15073199/10416112/e77f8c58-7007-11e5-832b-69f16b05ed95.png)
![code](https://cloud.githubusercontent.com/assets/15073199/10416113/ea8f2200-7007-11e5-8d53-82a2a7f4ed98.png)

**Comment by roncli** [comment]
Blizzard made the "realm" field a collection, which is mucking up the auctions.  I will put out a fix shortly.

**Comment by roncli** [comment]
Fixed in version 2.0.1!

Also, as a note, I created this library to interface with my guild website, six-minutes.com.  However, that guild is no longer in business, so I haven't kept it updated.  Since that was the main thing I used this library for, I haven't been keeping up with the library, either.

I am still good for supporting it, though, since I still use the Blizzard API in node.js, so if you (or anyone else!) runs across anything, just drop an issue, and I'll do my best to get to it! :)
